src/pages/examples.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { Composition } from 'atomic-layout'
+
+import Layout from '../components/layout'
+import { Grid } from '../components/Grid'
+import { ExampleItem } from '../components/exampleItem'
+import { TextLead } from '../components/TextLead'
+import { ReactComponent as GraphQLLogo } from '../images/graphql-logo.svg'
+import { ReactComponent as JavaScriptIcon } from '../images/logos/javascript-2.svg'
+import { ReactComponent as StorybookIcon } from '../images/logos/storybook.svg'
+import { Heading } from '../components/Heading'
+
+const ExampleList: React.FC = ({ children }) => {
+  return (
+    <Composition
+      gap={64}
+      templateColsMd="repeat(2, 1fr)"
+      templateColsLg="repeat(3, 1fr)"
+    >
+      {children}
+    </Composition>
+  )
+}
+
+const Examples = () => {
+  return (
+    <Layout>
+      <Grid paddingVertical={64} paddingVerticalMd={80}>
+        <Heading>Examples</Heading>
+        <TextLead>
+          Browser through the collection of Mock Service Worker usage examples
+          with various libraries and frameworks.
+        </TextLead>
+        <Composition gap={32} gapMd={80} marginTop={80}>
```

Comment:
Sorry, my bad, suggested `gap` changes in the wrong Composition. Please leave it as `gap={80}` always here. Thanks!